### PR TITLE
v3 - Eirini support

### PR DESCRIPTION
**Pull request by viovanov**

### CODE REVIEW

File: deploy/helm/scf/assets/operations/instance_groups/bits-service.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+- type: replace
+  path: /instance_groups/name=bits/jobs/name=bits-service/properties/bits-service/droplets/webdav_config/private_endpoint
+  value: *blobstore_url
+- type: replace
+  path: /instance_groups/name=bits/jobs/name=bits-service/properties/bits-service/packages/webdav_config/private_endpoint
+  value: *blobstore_url
+
+- type: replace
+  path: /instance_groups/name=bits/jobs/name=bits-service/properties/bits-service/private_endpoint
+  value: https://((deployment-name))-bits:4443
```
Comment (mook-as) [inline deploy/helm/scf/assets/operations/instance_groups/bits-service.yaml]: Isn't that port 443?

File: deploy/helm/scf/templates/eirini.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# A service for the cc uploader
```
Comment (f0rmiga) [inline deploy/helm/scf/templates/eirini.yaml]: This file should be rendered only when Eirini feature is enabled.

File: deploy/helm/scf/templates/ops.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@ metadata:
     helm.sh/chart: {{ include "scf.chart" .Root }}
 data:
   ops: |-
-    {{- .Root.Files.Get .Path | nindent 4 -}}
+    {{- tpl (.Root.Files.Get .Path) .Root | nindent 4 -}}
```
Comment (f0rmiga) [inline deploy/helm/scf/templates/ops.yaml]: +1

With this, I think we can stop creating lots of implicit vars.

File: dev/kind/start.sh [hunk 4]
```diff
@@ -25,6 +25,18 @@ export KUBECONFIG
 "${KUBECTL}" patch storageclass local-path \
   --patch '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true", "storageclass.beta.kubernetes.io/is-default-class":"true"}}}'
 
+# Deploy the kube dashboard.
+"${KUBECTL}" apply -f https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta1/aio/deploy/recommended.yaml
+
+# Create the metrics server.
+cd `mktemp -d`
+git clone --branch 'v0.3.3' https://github.com/kubernetes-incubator/metrics-server.git
```
Comment (f0rmiga) [inline dev/kind/start.sh]: This should be managed by Bazel.

File: deploy/helm/scf/templates/eirini.yaml (hunk 2 above)
Comment (viovanov) [reply deploy/helm/scf/templates/eirini.yaml]: 👍

File: deploy/helm/scf/templates/ops.yaml (hunk 3 above)
Comment (viovanov) [reply deploy/helm/scf/templates/ops.yaml]: Yes, very useful and seamless

File: deploy/helm/scf/assets/operations/instance_groups/bits-service.yaml [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+  value: *blobstore_url
+- type: replace
+  path: /instance_groups/name=bits/jobs/name=bits-service/properties/bits-service/droplets/webdav_config/private_endpoint
+  value: *blobstore_url
+- type: replace
+  path: /instance_groups/name=bits/jobs/name=bits-service/properties/bits-service/packages/webdav_config/private_endpoint
+  value: *blobstore_url
+
+# TODO: check to see where these should actually be public - what are the usecases?
+# A proper solution for eirini would be to
```
Comment (f0rmiga) [inline deploy/helm/scf/assets/operations/instance_groups/bits-service.yaml]: Tell me more! :P

**Pull request merged**